base/linalg/dense.jl

```diff
@@ -343,31 +345,16 @@ function ^{T}(A::AbstractMatrix{T}, p::Real)
         retmat = retmat * full(powm(UpperTriangular(float.(A)), real(p - floor(p))))
         d = diag(A)
     else
-        S,Q,d = schur(complex(A))
+        S,Q,d = schur(complex(full(A)))
```

Comment:
why shouldn't this let a specialized method of `schur` be called when there is one?